test/automated_test.py

```diff
@@ -99,7 +99,7 @@ def main():
     test_time = (stop - start)
 
     print_summary(test_list, result_list, test_time)
-    with open(summary_file, "wb") as output_file:
+    with open(summary_file, "w") as output_file:
```

Comment:
was this causing problems or just changed since it isn't a binary file?

Reply:
In Python 3, files open in binary mode only accept bytes-like objects in their `write()` method, and return `bytes` from `read()`.